src/js/pie.js

```diff
@@ -21,19 +21,25 @@ https://raw.githubusercontent.com/fluid-project/chartAuthoring/master/LICENSE.tx
     // 4. update the pie when the data set changes, including adding or removing slices
 
     fluid.defaults("floe.chartAuthoring.pieChart.pie", {
-        gradeNames: ["floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent",  "autoInit"],
+        gradeNames: ["floe.chartAuthoring.valueBinding","floe.chartAuthoring.totalRelaying", "floe.d3ViewComponent"],
```

Comment:
@amb26 - no worries. it sounds like this at least serendipitously surfaced an issue with the linting in the core framework. I was slightly confused because if there's one thing I'm sure of, it's that my linter for this project complains at me when I don't indent properly :) https://gist.github.com/waharnum/92ffbe9de590e164de30